app/controllers/roles_controller.rb

```diff
@@ -225,4 +230,36 @@ def count_only?
   def count_payload(dataset)
     { count: dataset.count }
   end
+
+  def audit_list_success
+    additional_params = %i[account count kind search limit offset]
+    options = params.permit(*additional_params)
```

Comment:
In this case, `permit` and `slice` accomplish the same outcome.  `to_h` will only include the parameters that have been permitted.  This allows us to simplify this to something like:

```suggestion
    options = params.permit(*additional_params).to_h.symbolize_keys
```

Reply:
@jvanderhoof Thanks on the review, can you please review again?